pkgs/os-specific/linux/nvidia-x11/settings.nix

```diff
@@ -104,19 +116,17 @@ stdenv.mkDerivation {
 
   nativeBuildInputs = [ pkg-config m4 addOpenGLRunpath ];
 
-  buildInputs = [ jansson libXv libXrandr libXext libXxf86vm libvdpau nvidia_x11 gtk2 dbus ]
+  buildInputs = [ jansson libXv libXrandr libXext libXxf86vm libvdpau nvidia_x11 dbus ]
+    ++ lib.optionals withGtk2 [ gtk2 ]
```

Comment:
I think this breaks `nvidia-settings` < [525.53](https://github.com/NVIDIA/nvidia-settings/commit/76b10f1b028cdccfa1b524955d9b8ca719c206c6#diff-3e2513390df543315686d7c85bd901ca9256268970032298815d2f893a9f0685), which is still included with the `legacy_470` nvidia driver. Before that version gtk2 was mandatory for the build.

It could be fixed by changing the default attribute to:
```nix
, withGtk2 ? lib.versionOlder nvidia_x11.settingsVersion "525.53"
```

Reply:
Good catch! My card is new enough that I didn't test the old version.